examples/hh/ppo_hh.py

```diff
@@ -77,7 +77,9 @@ def forward(self, input_ids):
         reward_model.load_state_dict(torch.load(checkpoint))
         reward_model.eval()
         reward_model.requires_grad_(False)
-        device = torch.cuda.device_count() - 1
+        device = os.environ.get("ACCELERATE_TORCH_DEVICE", None)
+        if device is None:
+            device = torch.cuda.device_count() - 1
```

Comment:
Regarding this particular example I wanted to ask: what would be the easiest way to hide one GPU so that it's accessible from the main rank, however is not included into DP group. For example here it's done simply with `accelerate --num_processes 7 ...` and 8th GPU would be serving a reward model, which is queried from the main rank, but would not participate in the training (it will OOM otherwise)

Reply:
hmm, it would probably require the rank 0 worker to be assigned 2 GPUs, but I am not sure if that's easily done with the APIs we have in Ray Train. let me dig into that!